Release/include/pplx/threadpool.h

```diff
@@ -46,6 +46,41 @@ template<class T>
 using java_local_ref = std::unique_ptr<typename std::remove_pointer<T>::type, java_local_ref_deleter>;
 #endif
 
+#if defined(_Win32)
+ // manually init Winsock
+template <int Major = 2, int Minor = 0>
+class winsock_init_helper
+{
+public:
+    winsock_init_helper()
+    {
+        if (InterlockedIncrement(&m_data.init_count) == 1)
```

Comment:
This ctor runs in DllMain and thus has the exact same behavior as that you're trying to inhibit.

Reply:
Why runs this in DllMain? I thouhgt this would run if an instance of threadpool will gets created. Is an instance of threadpool created in Dllmain?

Btw pushed a new version with fixed #if defined(_WIN32).

Reply:
Can you explain why this runs in dllMain?

Reply:
@BillyONeal can you give some detail about you concerns?